tests/manager/manager-executeBulkWrite_error-008.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+MongoDB\Driver\Manager::executeBulkWrite() and connection exception bubbling
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$bulk = new MongoDB\Driver\BulkWrite;
+$bulk->insert(['x' => 1]);
+
+// Invalid host cannot be resolved
+$manager = new MongoDB\Driver\Manager('mongodb://invalid.host:27017', ['serverSelectionTimeoutMS' => 1]);
+
+try {
+    $manager->executeBulkWrite(NS, $bulk);
+} catch (Exception $e) {
+    do {
+        printf("%s: %s\n", get_class($e), $e->getMessage());
+    } while ($e = $e->getPrevious());
+}
+
+echo "\n";
+
+// Valid host refuses connection
+$manager = new MongoDB\Driver\Manager('mongodb://localhost:54321', ['serverSelectionTimeoutMS' => 1]);
+
+try {
+    $manager->executeBulkWrite(NS, $bulk);
+} catch (Exception $e) {
+    do {
+        printf("%s: %s\n", get_class($e), $e->getMessage());
+    } while ($e = $e->getPrevious());
+}
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECT--
+MongoDB\Driver\Exception\RuntimeException: No suitable servers found (`serverselectiontryonce` set): [Failed connecting to 'invalid.host:27017': php_network_getaddresses: getaddrinfo failed: Name or service not known]
+MongoDB\Driver\Exception\ConnectionException: MongoDB\Driver\Manager::executeBulkWrite(): php_network_getaddresses: getaddrinfo failed: Name or service not known
+
+MongoDB\Driver\Exception\RuntimeException: No suitable servers found (`serverselectiontryonce` set): [Failed connecting to 'localhost:54321': Connection refused]
+===DONE===
```

Comment:
I believe that should all throw a ConnectionException. For HHVM, the diff with your expected output is:

```
001+ MongoDB\Driver\Exception\ConnectionTimeoutException: No suitable servers found (`serverselectiontryonce` set): [Failed to resolve 'invalid.host']
002+ 
003+ MongoDB\Driver\Exception\ConnectionTimeoutException: No suitable servers found (`serverselectiontryonce` set): [connection timeout calling ismaster on 'localhost:54321']
001- MongoDB\Driver\Exception\RuntimeException: No suitable servers found (`serverselectiontryonce` set): [Failed connecting to 'invalid.host:27017': php_network_getaddresses: getaddrinfo failed: Name or service not known]
002- MongoDB\Driver\Exception\ConnectionException: MongoDB\Driver\Manager::executeBulkWrite(): php_network_getaddresses: getaddrinfo failed: Name or service not known
003- 
004- MongoDB\Driver\Exception\RuntimeException: No suitable servers found (`serverselectiontryonce` set): [Failed connecting to 'localhost:54321': Connection refused]
```

I consider my output the right one to be honest.